src/orchestrator/csv/entry_16.csv

```diff
@@ -0,0 +1,17 @@
+username,balance_ETH_ETH,balance_USDT_ETH
```

Comment:
Can you move `csv` folder to the root of the project? I just think it's more clean to not have this within the source code of a module. In this case `orchestrator`

Reply:
Moved to root [c8a2d7f](https://github.com/summa-dev/summa-aggregation/pull/3/commits/c8a2d7f0ae2411b5e44a86837bc43dda5256f6be)